### PR TITLE
fix(File): Add HTMLFragment casting to $Tag / $getTag  - Github Issue #147

### PR DESCRIPTION
**Pull request by silbinarywolf**
fix(File): Add HTMLFragment casting to $Tag / $getTag  - Github Issue #147

### CODE REVIEW

**Comment by tractorcow** [comment]
If this is a bugfix should we put it into 4 or 4.1 / 4.2 branches?

**Comment by silbinarywolf** [comment]
Are you asking me? There are no 4.1/4.2 branches for this repo. It's using `1`, `1.1`, etc.

File: src/File.php [hunk 1]
```diff
@@ -157,7 +157,10 @@ class File extends DataObject implements AssetContainer, Thumbnail, CMSPreviewab
     );
 
     private static $casting = array (
-        'TreeTitle' => 'HTMLFragment'
+        'TreeTitle' => 'HTMLFragment',
+        'getTreeTitle' => 'HTMLFragment',
+        'Tag' => 'HTMLFragment',
+        'getTag' => 'HTMLFragment',
```
Comment (kinglozzer) [inline src/File.php]: Is `getTag` as well as `Tag` required? Perhaps we should fix that (separate to this PR) if so 😕

Comment (tractorcow) [reply src/File.php]: It should really just be the short name ('Tag') but it doesn't hurt I guess?

**Comment by maxime-rainville** [comment]
That went to the wrong branch. I should have merge it against `1`

**Comment by maxime-rainville** [comment]
I cherry picked the commit and did a PR against `1` #152.

**Pull request merged**